### ISSUE TITLE
issue with germline mutation hdf5 file in peptide generation script

### ISSUE THREAD

**Issue by varsha090597** [ratschlab/pancanatlas_code_public#5]
Hi,

I am trying to run the compute_peptide_lists.py script under the peptide generation directory. The germline mutations file the script takes in is in the hdf5 format. Since I had a vcf file I went ahead and converted it to hdf5 format using a tool called ipyrad (https://ipyrad.readthedocs.io/en/latest/API-analysis/cookbook-vcf2hdf5.html#). It looks from the utils_muatation.py script that the code has specific array names it looks for from the germline hdf5 file (like allele_alt, allele_ref) which the ipyrad tool doesn't use. 

Long story short, any guidance or help regarding how the germline mutations file was generated in hdf5 format, or if it was converted from a vcf file it would be helpful if you can supply that script, for me to be able to run the compute_peptide_lists.py script. 

Any other suggestions which would help me resolve this error would also be helpful.

Thanks.
<img width="939" alt="Screen Shot 2020-12-18 at 5 04 55 PM" src="https://user-images.githubusercontent.com/61675674/102665649-314aaa80-4153-11eb-9946-8f00b812fba2.png">
